cogeo_mosaic_tiler/handlers/app.py

```diff
@@ -94,29 +84,38 @@ def _create(
     minzoom = int(minzoom) if isinstance(minzoom, str) else minzoom
     maxzoom = int(maxzoom) if isinstance(maxzoom, str) else maxzoom
     min_tile_cover = (
-        float(min_tile_cover) if isinstance(min_tile_cover, float) else min_tile_cover
+        float(min_tile_cover) if isinstance(min_tile_cover, str) else min_tile_cover
     )
 
-    mosaicid = get_hash(body=body, version=mosaic_version)
-
+    mosaicid = get_hash(
+        body=body,
+        minzoom=minzoom,
+        maxzoom=maxzoom,
+        min_tile_cover=min_tile_cover,
+        tile_cover_sort=tile_cover_sort,
+        version=mosaic_version,
+    )
     try:
-        mosaic_definition = fetch_mosaic_definition(_create_path(mosaicid))
-    except ClientError:
+        with MosaicBackend(_create_mosaic_path(mosaicid), client=s3_client) as mosaic:
+            meta = mosaic.metadata
```

Comment:
So currently you support uploads only to S3?

Reply:
on your fork you just need to update `_create_mosaic_path` to default to dynamoDB. 

`create` and `add` shouldn't let the user decide where to store the data IMO

Reply:
Why not? It's fine to have a default but it's good to give people the choice

Reply:
not sure I want someone to create 100 tables ...

Reply:
Also the point of contributing is so that I don't maintain my own fork...